### PR TITLE
Add Cloudflare Workers component

### PR DESCRIPTION
**Pull request by pinkerton**
## What has been implemented?

This adds support for building Cloudflare Workers scripts with Serverless components, and includes an example.

Shoutout to @EverlastingBugstopper and @nirvik for all of their hard work on this  🎊 

## Steps to verify

```
export CLOUDFLARE_AUTH_KEY=YOUR_AUTH_KEY
export CLOUDFLARE_AUTH_EMAIL=YOUR_EMAIL_ADDRESS
cd examples/cloudflare-workers-example
```
Modify `serverless.yml` to include your `zoneId`

`components deploy`
`components remove`

## Todos:

* [ ] Write tests
* [x] Write / update documentation
* [x] Run Prettier

### CODE REVIEW

File: registry/s3-uploader/src/uploader.js [hunk 1]
```diff
@@ -19,8 +19,7 @@ module.exports.handler = (event, context, callback) => {
     Key: key,
     ContentType: parts[0].type
   }
-  s3
-    .putObject(params)
+  s3.putObject(params)
```
Comment (EverlastingBugstopper) [inline registry/s3-uploader/src/uploader.js]: this change happened when running the auto formatter

**Comment by pinkerton** [comment]
Thanks for the feedback @pmuens, really appreciated! 

We'll update this PR with the `inputs` and other changes, and will look into this after releasing the provider plugin.

**Comment by raeesbhatti** [comment]
I'm going to merge this for now. Credentials via `inputs` can be added later on.

**Comment by pinkerton** [comment]
Thanks, @raeesbhatti!

**Pull request merged**